manifests/plugin/foreman_gutterball_config.pp

```diff
@@ -0,0 +1,28 @@
+# foreman_gutterball config
+class katello::plugin::foreman_gutterball_config(
+    $foreman_user  = 'foreman',
+    $foreman_group = 'foreman',
+    $foreman_plugins_dir = '/usr/share/foreman/plugins',
```

Comment:
I don't think this is the right location for the file to get laid down. I believe it should be `/etc/foreman/settings.plugins.d/` . See https://github.com/theforeman/foreman/blob/develop/config/settings.rb#L14